numpy/doc/indexing.py

```diff
@@ -294,6 +294,13 @@
 np.array([[1,2]]) (shape (1,2)) that is broadcast with the index array
 to produce a resultant array of shape (3,2).
 
+This is equivalent to: ::
+
+ >>> y[np.array([[0],[2],[4]]),np.array([[1,2]])]
+ array([[ 1,  2],
+        [15, 16],
+        [29, 30]])
```

Comment:
```suggestion
In effect, the slice and index array operation are independent.
The slice operation extracts the 1st and 3rd columns in y,
followed by the index array operation which extracts the 0th, 2nd
and 4th rows from the sliced array.

This is equivalent to: ::

>>> y[:, 1:3][np.array([0, 2, 4]), :]
```

Reply:
Thanks, I applied your changes manually, since they span outside of what github can do with the suggest API...
@paulrougieux you used your "master" branch here, which means that these commits will have been pushed to your master branch. Once this is all finished, you will want to forceably reset your master to the NumPy master. Generally, sooner or later it is always painful when working directly on "master" for pull requests.